src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditAccountTests.java

```diff
@@ -1,20 +1,20 @@
 package com.wikia.webdriver.testcases.specialpagestests;
 
 import com.wikia.webdriver.common.contentpatterns.PageContent;
+import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.core.url.UrlBuilder;
 import com.wikia.webdriver.common.properties.Credentials;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
-import com.wikia.webdriver.elements.mercury.pages.login.SignInPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.AttachedSignInPage;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.editaccount.EditAccount;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.special.login.SpecialUserLoginPageObject;
 
 import org.testng.annotations.Test;
 
-@Test(groups = "EditAccountTest")
+@Test(groups = "EditAccountTest", enabled = false)
```

Comment:
these tests are never run on jenkins. After a quick investigation I found out they should be owned by Sus, filed https://wikia-inc.atlassian.net/browse/QAART-959 for that - let them handle the refactoring where needed